### DISCUSSION TITLE
Handling Uppercase PX in Tailwind with postcss-pxtorem Integration

### DISCUSSION THREAD

**Discussion by adolfheir** [tailwindlabs/tailwindcss#14280] [Help] [answered]
In my project, I use both postcss-pxtorem and Tailwind. One feature of postcss-pxtorem is that it does not automatically replace uppercase PX with rem. However, when writing uppercase PX in Tailwind, it gets recognized as a color by JIT. Can you tell me how to handle this case?

<img width="589" alt="image" src="https://github.com/user-attachments/assets/c3364c02-8bbc-4790-a7b4-2d27f5e750c9">
<img width="838" alt="image" src="https://github.com/user-attachments/assets/e16f3512-82cb-412a-b420-9ebba6895f4c">

**Comment by wongjn** [comment, marked as answer]
As per [the documentation](https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities):

> Many utilities in Tailwind share a common namespace but map to different CSS properties. For example `text-lg` and `text-black` both share the `text-` namespace, but one is for `font-size` and the other is for `color`.
>
> […]
>
> In these situations, you can “hint” the underlying type to Tailwind by adding a [CSS data type](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Types) before the value:
>
> ```html
> 
> <div class="text-[length:var(--my-var)]">...</div>
> 
> 
> <div class="text-[color:var(--my-var)]">...</div>
> ```

So for your case, you should be able to do:

```html
<div class="text-[length:13PX]">
```

**Reply by adolfheir** [reply]
Thank you I didn't see this way to set the data type before
